src/estimation/estimation_main.py

```diff
@@ -45,13 +46,16 @@ def run_estimation(
     # calculate the weights for outliers
     weighted_df = weights.outlier_weights(df)
 
+    # apply the weights to the dataframe and apply the specified rounding
+    estimated_df = appweights.apply_weights(weighted_df, 4)
+
     if config["global"]["output_estimation_qa"]:
         EstMainLogger.info("Outputting estimation QA file.")
         tdate = datetime.now().strftime("%Y-%m-%d")
         cell_qa_filename = f"estimation_weights_qa_{tdate}_v{run_id}.csv"
         full_qa_filename = f"full_estimation_qa_{tdate}_v{run_id}.csv"
         write_csv(f"{est_path}/estimation_qa/{cell_qa_filename}", qa_df)
-        write_csv(f"{est_path}/estimation_qa/{full_qa_filename}", weighted_df)
+        write_csv(f"{est_path}/estimation_qa/{full_qa_filename}", estimated_df)
```

Comment:
I took a look at the output and it looks good. It also takes care of ticket https://jira.ons.gov.uk/browse/RDRP-184 so i moved it to done